### ISSUE TITLE
Panel Spring mechanism

### ISSUE THREAD

**Issue by nabulator** [finalarcadia/openpad#4]
Your design indicated that you left 1.25 inches between the supports and the panels; are you simply using foam insulation / wooden popsicle sticks to raise the panel to the correct height? What spring mechanism did you choose for your final design? 

This is what I got from looking at the pictures: 
![panel_ddr-diagram](https://user-images.githubusercontent.com/6231643/40274348-72b5d97a-5ba2-11e8-84cf-42db6a8b0912.jpg)

You'd probably want something springy so the sensor can recover quickly before the next trigger.

**Comment by gabe-fz** [comment]
So the wood panel must touch the sensor. Either by raising the sensor or adding some risers on top of sensor (or both). There must be a small gap left between the bottom of the wood panel and any support below, so that the sensor is allowed to flex. Can be anything to stop the wood panel, doesnt have to be a spring (the movement is very little), but any rubber or similar material helps with sound dampening.

Currently working on a different design with 4 smaller load cells. Should be easier that way.

**Comment by nabulator** [comment]
So I ended up purchasing these threaded rubber feet and attaching them to sensor bar so you can adjust the distance and force to the panel itself. This is somewhat similar to the SMX pad design. It turns out to work pretty well. 

![image](https://user-images.githubusercontent.com/6231643/82738444-977a6680-9d05-11ea-9360-a33f96f7e09d.png)

I ended up using some crafting foam sheets and hot gluing them. 

I am really interested in the using 4 smaller load cells... I noticed that pad registration is rather localized. I would say the change in voltage is halved or more when moving away from the center of the load cell. Apparently, you can [hook up the sensors in parallel and the sensors will take the average of](https://electronics.stackexchange.com/questions/484863/connecting-4-load-cells-in-parallel) the force by itself! No extra circuitry or analog inputs needed. The SMX pads however, seem to sample each sensor and take the `max` of the inputs.

**Comment by gabe-fz** [comment]
Good idea with the rubber screws. I have by now tried the 4 load cells in parallel and can confirm that it works well. Just need to come up with a good and easy mounting method. Hoping that just resting a panel  freely on top of 4 load cells with rubber tops will work well. Also by this point the cables become kind of a mess, so hoping to learn some pcb design and get some prototypes done at some point.

**Comment by nabulator** [comment]
I've been drilling holes and tapping them, allowing me to use the same bolt to hold the sensor down. I suppose maybe you'd still want some support on the sides of the pad. I guess here's another MS paint design of what I was thinking. 
![openpad_panel_design](https://user-images.githubusercontent.com/6231643/82835844-32ed1200-9e93-11ea-9f1e-4e71a4842e90.png)
